EDDNResponder/EDDNResponder.cs

```diff
@@ -206,37 +206,23 @@ private void sendCommodityInformation()
                 {
                     foreach (CompanionAppEconomy economy in EDDI.Instance.CurrentStation.economies)
                     {
-                        EDDNEconomy eddnEconomy = new EDDNEconomy();
-                        eddnEconomy.name = economy.name;
-                        eddnEconomy.proportion = economy.proportion;
+                        EDDNEconomy eddnEconomy = new EDDNEconomy(economy);
                         eddnEconomies.Add(eddnEconomy);
                     }
                 }
 
                 List<EDDNCommodity> eddnCommodities = new List<EDDNCommodity>();
-                foreach (CommodityMarketQuote commodity in EDDI.Instance.CurrentStation.commodities)
+                foreach (CommodityMarketQuote quote in EDDI.Instance.CurrentStation.commodities)
```

Comment:
I still think we're making this more complex than it needs to be. In EDDI.cs, we generate the MarketInformationUpdated event from the ConditionallyRefreshProfile method, and in that method we also have access to `profile.station.commodities` Can we just add that same data to the MarketInformationUpdated event and pass it directly?

Reply:
The data is already processed by the CompanionServiceApp in CommodityQuotesFromProfile, which is not the problem... it worked fine before localization. If we broke the object with localization, we need to fix that part. Per the old saying, let's not throw the baby out with the bath water.

Reply:
I did think about this at length. The deciding factor for me was that the local `Market.json` file has differently cased keys -- "BuyPrice" etc. That means that if we were to start using it, a conversion mechanism of this kind would be needed. 

In general I like that we have classes such as `EDDNCommodity`. They decouple the EDDN API from our own representation, leaving us free to evolve our representation to suit our needs.